docs/analytics/apis/taxonomy-api.md

```diff
@@ -71,7 +70,7 @@ Content-Type: application/x-www-form-urlencoded
 category_name=Marketing
 ```
 
-##### Body parameters
+##### Request parameters
```

Comment:
We aren't making request via the body, but instead via the URL parameters. Referring to body here is confusing.